index.html

```diff
@@ -210,6 +214,18 @@
 
         const sortByLastName = (personList) => sortByFirstName(personList).reverse();
 
+        function filterPersonListByValidHeadshots(list){
+            return list.filter((person) => person.headshot.url != null );
+        }
+
+        function addFullNameProperty(list){
+            for (var x = 0; x < list.length; x++){
+                list[x].fullName = list[x].firstName + ' ' + list[x].lastName;
+            }
+            return list;
+        }
```

Comment:
add the full name property (first name + last name) to the person object on array initialization. Used in search function. Better to transform this property on the person object once in the beginning instead of doing it on the fly in the search function multiple times.